.github/workflows/e2e.yml

```diff
@@ -66,7 +66,6 @@ jobs:
     needs: build-frontend
     runs-on: ubuntu-latest
     container:
-      # renovate: datasource=docker depName=cypress/browsers versioning=docker
```

Comment:
Why did you remove the renovate update section for the cypress docker image?

Reply:
It doesn't work